### PR TITLE
198 unit tests naaccr etl

### PR DESCRIPTION
**Pull request by mgurley**

### CODE REVIEW

File: etl/naaccr_etl_postgresql.sql [hunk 1]
```diff
@@ -1,3 +1,4 @@
+SET search_path TO omop, public;
```
Comment (rtmill) [inline etl/naaccr_etl_postgresql.sql]: Intended?

Comment (mgurley) [reply etl/naaccr_etl_postgresql.sql]: no I will remove that.

File: etl/naaccr_etl_postgresql.sql [hunk 2]
```diff
@@ -305,7 +306,7 @@ CREATE TABLE naaccr_data_points_tmp
   SET naaccr_item_value =
 		CASE
 			WHEN CHAR_LENGTH(naaccr_item_value) != 8 THEN NULL
-			WHEN CASE WHEN (naaccr_item_value ~ '^([0-9]+\.?[0-9]*|\.[0-9]+)$') THEN 1 ELSE 0 END <> 1 THEN NULL
+			WHEN CASE WHEN (CAST(naaccr_item_value AS VARCHAR) ~ '^([0-9]+\.?[0-9]*|\.[0-9]+)$') THEN 1 ELSE 0 END <> 1 THEN NULL
```
Comment (rtmill) [inline etl/naaccr_etl_postgresql.sql]: We need to cast this?

**Comment by dsonnetiqvia** [comment]
approve

**Pull request merged**